frontend/routes/components/textField/textField.component.tsx

```diff
@@ -102,7 +107,7 @@ export class TextField extends React.PureComponent<IProps, IState> {
 	}
 
 	get fieldValue() {
-		return this.props.requiredConfirm ? this.state.currentValue : this.props.value;
+		return this.props.requiredConfirm ? this.state.currentValue : this.props.value as string;
```

Comment:
Is the string cast this necessary? I'm a bit dubious since we never had it previously.

Reply:
This is because the `copy` function from` copy-to-clipboard` expects this value to be a string, but it is not certain because props `value` declared in BaseTextFieldProps interface is an unknown type.
However, we can redeclare it in our own IProps interface as an optional string, and it seems to do the trick, I don't think we should expect anything else than string here.